### PR TITLE
2021 09 08 authenticator code match and context tristate

### PR DESCRIPTION
**Pull request by mjsantani**
Creating pull request to delete old feature settings and shownContext flags and replacing with new tristate flags.

### CODE REVIEW

**Review by andres-canello — APPROVED** [review]
Made a small fix

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -8804,6 +8804,39 @@
         "CreatedDateTime":"2021-08-27T04:29:52.029Z",
         "WorkloadArea":"Identity and access",
         "SubArea":"Identity and sign-in"
-    }
+    },
+    {"ChangeList": [
+      {
+        "Id":"afcc706e-10c5-11ec-bd5a-a13d1ed8c60f",
```
Comment (Lauragra) [inline changelog/Microsoft.DirectoryServices.json:8810]: Please make sure that the IDs within a Changelist item are the same; we shouldn't have a different ID for each item. Thanks.

Comment (mjsantani) [reply changelog/Microsoft.DirectoryServices.json]: updated the changelog with the right IDs and added a couple missing items as well.

**Comment by mjsantani** [comment]
@Lauragra i've addressed the comments. Can we get a fresh review please. Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**